src/python/tests/genbank/test_download.py

```diff
@@ -0,0 +1,92 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.io.genomio.genbank.download` module.
+
+The unit testing is divided into one test class per submodule/class found in this module, and one test method
+per public function/class method.
+
+Typical usage example::
+    $ pytest test_download.py
+
+"""
+
+from pathlib import Path
+from typing import Any
+
+import pytest
+from pytest import raises
+from unittest.mock import patch, MagicMock
+
+from ensembl.io.genomio.genbank.download import download_genbank, DownloadError
+
+
+@pytest.mark.parametrize(
+    "accession",
+    [
+        ("CM023531.1"),
+    ], 
+)
+@patch('ensembl.io.genomio.genbank.download.requests.get')
+class TestDownloadGenbank:
+    """Tests for the `download_genbank` class"""
+    
+    def test_successful_download(self, mock_requests_get: Any, tmp_dir: Path, accession: str) -> None:
```

Comment:
Based on the documentation, it seems that `mock_requests_get` should be a `Mock` instance:
```suggestion
    def test_successful_download(self, mock_requests_get: Mock, tmp_dir: Path, accession: str) -> None:
````
_Note:_ remember to add `from unittest.mock import Mock` at the top.